grammar.js

```diff
@@ -930,12 +930,12 @@ module.exports = grammar({
     ),
 
     string: $ => seq(
-      field('prefix', alias($._string_start, '"')),
-      repeat(choice(
-        field('interpolation', $.interpolation),
-        field('string_content', $.string_content)
-      )),
-      field('suffix', alias($._string_end, '"'))
+      field('open', alias($._string_start, '"')),
```

Comment:
You are right that the `$._string_start` is not a prefix of the _content_ of the string, but I don't think anyone would be confused into thinking it was.

The term "prefix" _is_ used in the official Python documentation for the bit that goes before the opening single/double quotes (and so, the `prefix` field here at least _contains_ what most people familiar with Python nomenclature would consider to be the prefix). I think ideally we would want to separate the string into to all of its constituent parts: the prefix, the opening quotes, the content, and the ending quotes.

This would require some changes to the scanner, however, and in practice it's easy to split the contents of the `prefix` field into the prefix and starting quotes.

Reply:
> You are right that the $._string_start is not a prefix of the content of the string, but I don't think anyone would be confused into thinking it was.
As I mentioned above it would be better in the future to assign to the `prefix` term a correct sematic.